### PR TITLE
docs: update creating autoware repositories page

### PR DESCRIPTION
**Pull request by ismetatabay**
## Description

This PR add more information about creating autoware meta-repository and other individual repositories. (autoware_launch, autoware_individual_params, sample_vehicle_launch and sample_sensor_kit_launch)

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+A meta-repository is a repository that manages multiple repositories, and [Autoware](https://github.com/autowarefoundation/autoware) is one of them.
+It serves as a centralized control point for referencing, configuring,
+and versioning other repositories.
+In order to do that,
+Autoware meta-repository includes [`autoware.repos`](https://github.com/autowarefoundation/autoware/blob/main/autoware.repos) file for managing multiple repositories.
+We will use [VCS](https://github.com/dirk-thomas/vcstool) tool (Version Control System) for handling .repos file.
+VCS provides us with the capability to import, export, and pull from multiple repositories.
+
+We will use VCS for importing all necessary repositories in our workspace.
+Please follow the documentation for VCS and .repos file usage.
```
Comment (danielsanchezaran) [inline docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md:14]: Using Grammarly and ChatGPT I came up with a couple fixes and changes to this paragraph to improve its grammar and try to make it's text more concise. What do you think?
```suggestion
A meta-repository is a repository that manages multiple repositories, and [Autoware](https://github.com/autowarefoundation/autoware) is one of them. It serves as a centralized control point for referencing, configuring, and versioning other repositories. To accomplish this, the Autoware meta-repository includes the [`autoware.repos`](https://github.com/autowarefoundation/autoware/blob/main/autoware.repos) file for managing multiple repositories.
We will use the [VCS](https://github.com/dirk-thomas/vcstool) tool (Version Control System) to handle the .repos file. VCS provides us with the capability to import, export, and pull from multiple repositories.
VCS will be used to import all the necessary repositories to build Autoware into our workspace. Please refer to the documentation for VCS and .repos file usage.
```

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+## How to create and customize your autoware meta-repository
+
+### 1. Create autoware meta-repository
+
+If you want to integrate Autoware into your vehicle, the first step is to create an Autoware meta-repository.
+
+One easy way is to fork autoware repository and clone it.
+(For how to fork a repository, refer to [GitHub Docs](https://docs.github.com/en/get-started/quickstart/fork-a-repo))
```
Comment (danielsanchezaran) [inline docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md:23]:
```suggestion
One easy way is to fork the Autoware repository and clone it. (For instructions on how to fork a repository, refer to [GitHub Docs](https://docs.github.com/en/get-started/quickstart/fork-a-repo))
```

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+- In order to do that,
+  please go to the [autoware](https://github.com/autowarefoundation/autoware) repository
+  and click the fork button.
+  For example,
+  we will integrate `tutorial_vehicle` for these guidelines
+  (If you set up multiple types of vehicles,
+  adding a suffix like `autoware.vehicle_A` or `autoware.vehicle_B` is recommended),
+  so our forking should be like this:
+
+![forking-autoware_repository.png](images/forking-autoware_repository.png)
```
Comment (danielsanchezaran) [inline docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md:34]:
```suggestion
-In this guide, Autoware will be integrated into a `tutorial_vehicle` (Note: when setting up multiple types of vehicles, adding a suffix like `autoware.vehicle_A` or `autoware.vehicle_B` is recommended).
For the first step, please visit the [autoware](https://github.com/autowarefoundation/autoware) repository and click the fork button. The fork process should look like this:

![forking-autoware_repository.png](images/forking-autoware_repository.png)
```

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+
+For example, it should be for our documentation:
+
+```bash
+git clone https://github.com/leo-drive/autoware.tutorial_vehicle.git
+```
+
+#### 1.1 Create vehicle individual repositories
+
+For integrating autoware on your own individual vehicles, you need fork and modify the following repositories as well:
```
Comment (danielsanchezaran) [inline docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md:50]:
```suggestion

To integrate Autoware into your individual vehicles, you need to fork and modify the following repositories as well:
```

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+
+- [sample_sensor_kit](https://github.com/autowarefoundation/sample_sensor_kit_launch): This repository will be used for sensing launch files, their pipelines-organizations and sensor descriptions.
+  Please fork and rename as autoware meta-repository. At this point, our forked repository name will be `tutorial_vehicle_sensor_kit_launch`.
+- [sample_vehicle_launch](https://github.com/autowarefoundation/sample_vehicle_launch): This repository will be used for vehicle launch files, vehicle_descriptions and vehicle_model.
+  Please fork and rename this repository as well. At this point, our forked repository name will be `tutorial_vehicle_launch`.
+- [autoware_individual_params](https://github.com/autowarefoundation/autoware_individual_params): This repository stores parameters that change depending on each vehicle (i.e. sensor calibrations). Please fork
+  and rename this repository as well, our forked repository name will be `tutorial_vehicle_individual_params`.
```
Comment (danielsanchezaran) [inline docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md:57]:
```suggestion
- [autoware_individual_params](https://github.com/autowarefoundation/autoware_individual_params): This repository stores parameters that change depending on each vehicle (i.e. sensor calibrations). Please fork
  and rename this repository as well; our forked repository name will be `tutorial_vehicle_individual_params`.
```

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+- [autoware_launch](https://github.com/autowarefoundation/autoware_launch): This contains node configurations and their parameters for Autoware. Please fork
+  and rename this repository as previous forked repositories, our forked repository name will be `autoware_launch.tutorial_vehicle`.
+
```
Comment (danielsanchezaran) [inline docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md:60]:
```suggestion
- [autoware_launch](https://github.com/autowarefoundation/autoware_launch): This repository contains node configurations and their parameters for Autoware. Please fork and rename this repository as previously forked repositories; our forked repository name will be `autoware_launch.tutorial_vehicle`.

```

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md [hunk 7]
```diff
@@ -0,0 +1,166 @@
+# Creating Autoware repositories
+
+## What is Meta-repository?
```
Comment (danielsanchezaran) [inline docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md:3]:
```suggestion
## What is a Meta-repository?
```

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md [hunk 8]
```diff
@@ -0,0 +1,166 @@
...
+  and rename this repository as previous forked repositories, our forked repository name will be `autoware_launch.tutorial_vehicle`.
+
+### 2. Customize your autoware.repos for your environment
+
+You need to customize `autoware.repos` for your own vehicle's Autoware
+(It is included under the forked Autoware meta-repository directory).
+The all necessary autoware repos (except calibration and simulator repos)
+information is included in this file.
+
+Now we can add forked individual repositories to `autoware.repos` before the importing operation.
```
Comment (danielsanchezaran) [inline docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md:68]:
```suggestion
### 2. Customize autoware.repos for your environment

You need to customize your `autoware.repos` to import your forked repositories. `autoware.repos` usually includes information for all necessary Autoware repositories (except calibration and simulator repositories). Therefore, your forked repositories should also be added to this file.
```

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md [hunk 9]
```diff
@@ -0,0 +1,166 @@
...
+
+#### 2.1 Adding individual repos to autoware.repos
+
+After the forking of all repositories, we can start adding these repos to autoware.repos file.
+In order to do that, you should open autoware.repos via any text editor.
+You need to update `sample_sensor_kit_launch`,
+`sample_vehicle_launch`,
+`autoware_individual_params` and `autoware launch` with your own individual repos.
+In example at this tutorial,
+the necessary changes for our forked `tutorial_vehicle` repos should be like this:
```
Comment (danielsanchezaran) [inline docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md:78]:
```suggestion
#### 2.1 Adding individual repos to autoware.repos

After forking all repositories, you can start adding them to your Autoware meta-repository by opening the `autoware.repos` file using any text editor and updating `sample_sensor_kit_launch`, `sample_vehicle_launch`, `autoware_individual_params` and `autoware launch` with your own individual repos.
For example, in this tutorial, the necessary changes for our forked `tutorial_vehicle` repositories should be as follows:
```

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+  -   version: main
+  + launcher/autoware_launch.tutorial_vehicle:
+  +   type: git
+  +   url: https://github.com/leo-drive/autoware_launch.tutorial_vehicle.git
+  +   version: main
+  ```
+
+Please do same similar changes on your own autoware.repos file.
+After the changing these repositories,
+we will be ready to use VCS for importing all necessary repositories to our autoware workspace.
```
Comment (danielsanchezaran) [inline docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md:134]:
```suggestion
Please make similar changes to your own autoware.repos file. After making these changes, you will be ready to use VCS to import all the necessary repositories into your Autoware workspace.
```

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md [hunk 11]
```diff
@@ -0,0 +1,166 @@
...
+
+Firstly, we will create src directory under the own autoware meta-repository directory:
```
Comment (danielsanchezaran) [inline docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md:136]:
```suggestion
First, create a src directory under your own Autoware meta-repository directory:
```

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md [hunk 12]
```diff
@@ -0,0 +1,166 @@
...
+
+```bash
+cd <YOUR-AUTOWARE-DIR>
+mkdir src
+```
+
+Then, we will import all necessary repositories via:
```
Comment (danielsanchezaran) [inline docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md:143]:
```suggestion
Then, import all necessary repositories via:
```

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md [hunk 13]
```diff
@@ -0,0 +1,166 @@
...
+
+```bash
+cd <YOUR-AUTOWARE-DIR>
+vcs import src < autoware.repos
+```
+
+After the running `vcs import` command,
+all autoware repositories will be cloned in `src` folder under the Autoware directory.
```
Comment (danielsanchezaran) [inline docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md:151]:
```suggestion
After the running `vcs import` command, all autoware repositories will be cloned in the `src` folder under the Autoware directory.
```

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md [hunk 14]
```diff
@@ -0,0 +1,166 @@
...
+
+So, we can build our own repository with colcon build:
```
Comment (danielsanchezaran) [inline docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md:153]:
```suggestion
Now, you can build your own repository with colcon build:
```

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md [hunk 15]
```diff
@@ -0,0 +1,166 @@
...
+
+```bash
+cd <YOUR-AUTOWARE-DIR>
+colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release
+```
+
+Please refer to the following documentation link for instructions on how to create and customize each `vehicle_interface`:
+
```
Comment (danielsanchezaran) [inline docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md:161]:
```suggestion
Please refer to the following documentation links for instructions on how to create and customize each of your vehicle's packages:
```

**Comment by danielsanchezaran** [comment]
Overall, I feel the contents related to the "Create Autoware Meta-repository Guide" are very good nd easy to follow, I have made some suggestions to improve grammar and readability with some help from chatGPT @ismetatabay

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md (hunk 4 above)
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md]: done :+1:

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md (hunk 3 above)
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md]: done :+1:

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md (hunk 1 above)
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md]: done :+1:

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md (hunk 2 above)
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md]: done :+1:

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md (hunk 5 above)
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md]: done :+1:

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md (hunk 6 above)
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md]: done :+1:

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md (hunk 7 above)
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md]: done :+1:

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md (hunk 8 above)
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md]: done :+1:

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md (hunk 9 above)
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md]: done :+1:

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md (hunk 10 above)
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md]: done :+1:

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md (hunk 11 above)
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md]: done :+1:

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md (hunk 12 above)
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md]: done :+1:

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md (hunk 13 above)
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md]: done :+1:

File: docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md (hunk 15 above)
Comment (ismetatabay) [reply docs/how-to-guides/integrating-autoware/creating-your-autoware-repositories/creating-autoware-repositories.md]: done :+1:

**Comment by ismetatabay** [comment]
@danielsanchezaran, Thank you very much for your review comments. I completely agree with these changes, and I have implemented all of your suggestions. Consequently, I believe the code's readability and understandability have been enhanced :+1:

**Comment by ismetatabay** [comment]
@danielsanchezaran, @mitsudome-r Hi, is it okay to merge?

**Comment by danielsanchezaran** [comment]
LGTM

**Review by mitsudome-r — APPROVED** [review]

**Pull request merged**